### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by serenashe**
Signed-off-by: serenashe <serenashe@kubesphere.io>
Hi there, I fixed some typos in the documents and blogs. Please take a look, thanks.
Fix: #2206 

/assign @Felixnoo @Patrick-LuoYu

### CODE REVIEW

File: content/zh/docs/release/release-v321.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ weight: 18099
 - 修复**编辑项目配额**对话框中可用企业空间配额显示不正确的问题。（[#2613](https://github.com/kubesphere/console/pull/2613)，[@weili520](https://github.com/weili520)）
 - 修复**监控**对话框中时间范围选择下拉列表功能不正常的问题。（[#2722](https://github.com/kubesphere/console/pull/2722)，[@weili520](https://github.com/weili520)）
 - 修复部署创建页面可用配额显示不正确的问题。（[#2668](https://github.com/kubesphere/console/pull/2668)，[@weili520](https://github.com/weili520)）
-- 将文档地址更改为 [kubesphere.io](http://kubesphere.io/) 和 [kubespehre.com.cn](http://kubesphere.io/)。（[#2628](https://github.com/kubesphere/console/pull/2628)，[@weili520](https://github.com/weili520)）
+- 将文档地址更改为 [kubesphere.io](http://kubesphere.io/zh/) 和 [kubesphere.com.cn](https://kubesphere.com.cn/)。（[#2628](https://github.com/kubesphere/console/pull/2628)，[@weili520](https://github.com/weili520)）
```
Comment (Felixnoo) [inline content/zh/docs/release/release-v321.md:37]: I think `http://kubesphere.io/zh/` should be `https://kubesphere.io/`?

Comment (serenashe) [reply content/zh/docs/release/release-v321.md]: Changed. Thanks!

**Pull request merged**